ccmlib/scylla_node.py

```diff
@@ -1571,9 +1580,22 @@ def run_scylla_sstable(self, command, additional_args=None, keyspace=None, dataf
             additional_args = []
 
         scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
-        sstables = self._Node__gather_sstables(datafiles, keyspace, column_families)
         ret = {}
 
+        if datafiles is None and keyspace is not None:
+            tag = "sstable-dump-{}".format(uuid.uuid1())
+            kts = ",".join([f"{keyspace}.{column_family}" for column_family in column_families])
+            self.debug(f"run_scylla_sstable(): creating snapshot with tag {tag} to be used for sstable dumping")
+            self.nodetool(f"snapshot -t {tag} {kts}")
+            sstables = []
+            for column_family in column_families:
+                sstables.extend(glob.glob(os.path.join(self.get_path(), 'data', keyspace, f"{column_family}-*/snapshots/{tag}/*-Data.db")))
```

Comment:
You could use Pathlib to do this, it has some differences compared to glob (https://docs.python.org/3/library/pathlib.html#comparison-to-the-glob-module) though, not sure if they are relevant here
```
path = self.get_path() / "data" / "keyspace
path.glob(f"{column_family}-*/snapshots/{tag}/*-Data.db")
```

Reply:
Is there an advantage of using `Pathlib` instead of `glob` for this?

Reply:
Functionally no, I just consider it to be a better API as it is more concise and object oriented, so it is just a personal preference. Feel free to keep it if you like this better.

Reply:
I'd prefer to keep `glob`, it is the one I'm familiar with.